### PR TITLE
space-not-found

### PR DESCRIPTION
**Pull request by ethedev**

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -219,3 +219,18 @@ textarea {
     color: var(--header-bg);
   }
 }
+
+.block-container {
```
Comment (bonustrack) [inline src/style.scss:223]: We don't need to create new style for this, it's better to use primer-css classes.

File: src/views/Home.vue [hunk 2]
```diff
@@ -47,13 +48,21 @@
...
-              <div class="">
+              <div>
                 <h3 v-text="space.name" />
                 <div class="text-gray">{{ space.symbol }}</div>
               </div>
             </Block>
           </div>
         </router-link>
+
+        <div
```
Comment (bonustrack) [inline src/views/Home.vue:59]: You can use <Block> component here instead of <div> with the classes

Comment (jxdv) [reply src/views/Home.vue:59]: yes we can work on that separately later, here i havent done anything other than just removing the forgotten `class=""` text

File: src/style.scss (hunk 1 above)
Comment (jxdv) [reply src/style.scss:223]: yes well, this was `style="height: 250px; margin-bottom: 24px !important;"` before and so we need to switch it to non inline css, also not everything we can have it in primecss

File: src/style.scss [hunk 3]
```diff
@@ -224,3 +224,18 @@ textarea {
...
+  height: 250px;
+  margin-bottom: 24px !important;
+}
+
+.no-space {
```
Comment (bonustrack) [inline src/style.scss:233]: We don't need a custom class here you can use the class "p-4" and "text-align" instead.

**Comment by bonustrack** [comment]
Closing this PR, it's not relevant anymore.

**Comment by bonustrack** [comment]
Closing this again :), sorry bro, it was implemented in another PR.

**Pull request closed without merging**